index2.html

```diff
@@ -11,233 +11,99 @@
 </head>
 
 <body>
-	<!--//headedrr-->
-	<div class="container_body">
-		<header class="header">
-			<div class="header__inner">
-				<svg class="futerSVG">
-					<use xlink:href="sprite.svg#futer"></use>
-				</svg>
-			</div>
-			<nav class="navigation">
-				<a href="#" class="nav__link">Stays</a>
-				<a href="#" class="nav__link"> Attractions </a>
-				<a href="#" class="nav__link"> <svg class="NightSVG">
-						<use xlink:href="sprite.svg#Night-svg"></use>
-					</svg></a>
-				<a href="#" class="nav__link"> <svg class="CircleSVG">
-						<use xlink:href="sprite.svg#Circle-svg"></use>
-					</svg></a>
+	<!--The main wrapper of the first page of the project-->
+	<div class="wrapper">
+		<!--navigation and company logo-->
+		<header class="header_logo_navigation">
+			<img width="0" height="30" class="header_company logo" src="LinkedSprite.svg#src-2" alt="Company `s logo">
+			<!--header navigation-->
+			<nav class="header_navigation">
+				<a href="#" class="header_link" id="link_nav_stays">Stays</a>
+				<a href="#" class="header_link" id="link_nav_attractions">Attractions</a>
+				<a href="#" class="header_link night">
+					<img class="header_icon night" width="25" height="27" src="LinkedSprite.svg#src-3">
+				</a>
+				<a href="#" class="header_link circle ">
+					<img class="header_icon circle" width="35" height="35" src="LinkedSprite.svg#src-4">
```

Comment:
I advise you to abandon inline styles in favor of classes. I also recommend renaming the night and the circle to something more informative